Sources/RadioButton.swift

```diff
@@ -57,6 +80,8 @@ import UIKit
         addConstrainedSubview(selectedCenterView, constrain: .topMargin, .bottomMargin, .leftMargin, .rightMargin)
         selectedCenterView.layoutMargins = .zero
         selectedColor = { selectedColor }()
+        selectedRingColor = { selectedRingColor }()
+        normalRingColor = { normalRingColor }()
```

Comment:
I prefer to add the other properties here too:

```swift
        multilineLabels = { multilineLabels }()
        itemStackViewAlignment = { itemStackViewAlignment }()
```

(instead of duplicating the logic down in `RadioGroupItem.commonInit()`)